frontend/src/redux/filterSlice.ts

```diff
@@ -39,6 +43,14 @@ export function filterResource(
     return false;
   }
 
+  let status: string = '';
+  let phase: string = '';
+  if (isKubeCRD(item)) {
+    const kubeCRD: KubeCRD = item as KubeCRD;
+    status = String(kubeCRD?.status);
```

Comment:
I'm not sure if this was intentional, or maybe the isKubeCRD(item) guard covers this case? When it's undefined the result is... `String(undefined) -> 'undefined'`.

If it's a problem, an alternative would be something like:
`status = kubeCRD?.status ? kubeCRD?.status : '';`